### DISCUSSION TITLE
Minecraft Bedrock Cursor Problem

### DISCUSSION THREAD

**Discussion by Avisator1** [PlayCover/PlayCover#139] [Q&A]
I recently had MCPE decrypted by a friend on mine, and right when I opened the IPA I noticed the cursor kept disappearing when I open my inventory and rearrange it. I messaged a couple friends and I believe its because of Play Tools making the crosshair disappear when you open the app. Please Fix This!

**Comment by ZhichGaming** [comment]
Did you try pressing option to toggle the keymapping?

**Reply by Avisator1** [reply]
Yes, but it still doesn't work. I dont have OPTION on my keyboard, because its 60% but I think alt is the same thing

**Reply by ZhichGaming** [reply]
Try restarting your computer and if that doesn't work, please record a short video of the bug happening.

**Reply by Avisator1** [reply]
Ok here it is https://streamable.com/zxjsmm
Basically when I open my inventory I cant see my crosshair. I am clicking alt a bunch of times a nd I recently restarted the computer.

**Reply by ZhichGaming** [reply]
I see. The problem is that minecraft has a built in keyboard integration but it does not show the cursor properly. You can try going in playcover, right click on minecraft, app settings and disable key mapping.

**Reply by Avisator1** [reply]
I already tried that

**Reply by ZhichGaming** [reply]
I think that you will need to wait for our devs to implement a fix then.
